isovar/dataframe_builder.py

```diff
@@ -133,6 +146,9 @@ def add(self, variant, element):
                 name = self.rename_dict[name]
             self.columns_dict[name].append(value)
 
+        for column_name, fn in self.extra_column_fns.items():
+            self.columns_dict[column_name] = fn(variant, element)
```

Comment:
That was the original motivation for `extra_column_fn` but then I ended up adding it everywhere. Any idea for reducing repetition?